packages/wallets/src/evm/connectors/smart-wallet/index.ts

```diff
@@ -556,7 +576,9 @@ export class SmartWalletConnector extends Connector<SmartWalletConnectionArgs> {
       this.accountApi.createUnsignedUserOp(
         this.aaProvider.httpRpcClient,
         tx,
-        batchData,
+        {
+          batchData
```

Comment:
will also need the gasless flag here. the estimation would be different if its gasless vs not, might as well let users pass the config to estimateX

Reply:
Ohh, thought gas usage is mostly the same. Will update